src/Rules/PHPUnit/AssertSameWithCountRule.php

```diff
@@ -36,11 +36,19 @@ public function processNode(Node $node, Scope $scope): array
 		if (
 			$right instanceof Node\Expr\FuncCall
 			&& $right->name instanceof Node\Name
-			&& strtolower($right->name->toString()) === 'count'
 		) {
-			return [
-				'You should use assertCount($expectedCount, $variable) instead of assertSame($expectedCount, count($variable)).',
-			];
+			$method = strtolower($right->name->toString());
+			$message = 'You should use assertCount($expectedCount, $variable) instead of assertSame($expectedCount, %s($variable)).';
+
+			if ($method === 'count') {
+				return [
+					sprintf($message, 'count'),
```

Comment:
What about `sprintf($message, $method)` instead of this if/else?

Reply:
Shouldn’t we verify if the function is `count` or `sizeof`? I followed the if that was here 😁

Reply:
OK, just try to simplify the code somehow :)

Reply:
Okay :man_technologist: